src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/commands.py

```diff
@@ -31,6 +32,22 @@ def __call__(self, poller):
                     raise e
             return None
 
+    # iot hub certificate commands
+    cli_command(__name__, 'iot hub certificate list', custom_path.format('iot_hub_certificate_list'),
+                certificate_factory)
+    cli_command(__name__, 'iot hub certificate show', custom_path.format('iot_hub_certificate_get'),
+                certificate_factory)
+    cli_command(__name__, 'iot hub certificate create', custom_path.format('iot_hub_certificate_create'),
+                certificate_factory)
+    cli_command(__name__, 'iot hub certificate delete', custom_path.format('iot_hub_certificate_delete'),
+                certificate_factory)
+    cli_command(__name__, 'iot hub certificate generate-verification-code',
+                custom_path.format('iot_hub_certificate_gen_code'), certificate_factory)
+    cli_command(__name__, 'iot hub certificate verify', custom_path.format('iot_hub_certificate_verify'),
+                certificate_factory)
+    cli_command(__name__, 'iot hub certificate update', custom_path.format('iot_hub_certificate_update'),
```

Comment:
This should be a generic update.

Reply:
What do you mean by "generic update"?

Reply:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#generic-update-commands

Reply:
Gotcha, is this a valid scenario for this generic update command? In this case, the Certificate object is created by providing a friendly name and a certificate string. Once it's created the ONLY thing that can be updated is the entire certificate (meaning uploading an entirely new certificate string). The name cannot be 'updated' nor can any of the properties. Please correct me if I'm wrong, but does the generic update command simply provide a way to craft your update to an object before writing it back to the server?

Reply:
It wasn't completely clear to me from the description in the doc.